scripts/code_style.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
+"""Check or fix the code style by running Uncrustify.
```

Comment:
[blocker] We aren't enforcing this yet, which may confuse people outside the team. Please add a sentence that makes it clear that this is the future code style.